src/components/App/App.tsx

```diff
@@ -57,7 +60,7 @@ const App = (): JSX.Element => {
         return articleA.sentiment - articleB.sentiment;
       })
     } else {
-      sortedArticles = articles.slice().sort((articleA, articleB) => 0.5 - Math.random());
+      sortedArticles = articles.slice().sort(() => 0.5 - Math.random());
```

Comment:
Nice catch!

Reply:
Yeah, wasn't sure it would work without those params, but it does.